jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java

```diff
@@ -289,4 +294,36 @@ protected void addSchemaLabels(BuildConfiguration.BuildConfigurationBuilder buil
             buildBuilder.labels(labels);
         }
     }
-}
+
+    public void checkAndWarnIfProjectHasNotBeenBuilt() {
+      File finalOutputArtifact = null;
+      try {
+        finalOutputArtifact = JKubeProjectUtil.getFinalOutputArtifact(getProject());
```

Comment:
If there is an unexpected error here, you should ignore and return in my opinion